docs/reference/cluster/allocation-explain.asciidoc

```diff
@@ -35,12 +43,12 @@ specified as either the node id or node name.
 
 [source,js]
 --------------------------------------------------
-$ curl -XGET 'http://localhost:9200/_cluster/allocation/explain' -d '{
+{
```

Comment:
I'd make it a CONSOLE blob and explicitly script the test with `// TEST[skip:don't know the name or id of a node currently in the cluster]`. You could probably get one the same way we do with yaml tests but that seems like a lot of work. Maybe worth it if we find this snippet has an error, but otherwise not. This way we get the `COPY AS CURL` and `VIEW IN CONSOLE` links which are handy, even if we do skip the test.